Python/pylifecycle.c

```diff
@@ -1136,11 +1136,12 @@ init_interp_main(PyThreadState *tstate)
             return _PyStatus_ERR("can't initialize signals");
         }
 
-        if (_PyTraceMalloc_Init(config->tracemalloc) < 0) {
-            return _PyStatus_ERR("can't initialize tracemalloc");
+        if (config->tracemalloc) {
+           if ( _PyTraceMalloc_Start(config->tracemalloc) < 0) {
```

Comment:
```suggestion
           if (_PyTraceMalloc_Start(config->tracemalloc) < 0) {
```
(drive-by style nit)